### ISSUE TITLE
Questions about evaluation on VITS

### ISSUE THREAD

**Issue by GongBaoJiDingMan** [facebookresearch/Mask2Former#63]
It seems the eval for VITS is not performed during training as there is no annotations in json file
`# Test set json files do not contain annotations (evaluation must be performed using the COCO evaluation server).
 self._do_evaluation = "annotations" in self._ytvis_api.dataset`

So is it OK if I offer a json file with annotation in it?

**Comment by bowenc0221** [comment]
Yes, if you have annotation for the validation set you can evaluate it during training.
